addon/ng2/blueprints/ng2/files/angular-cli-build.js

```diff
@@ -7,11 +7,11 @@ module.exports = function(defaults) {
     vendorNpmFiles: [
       'systemjs/dist/system-polyfills.js',
       'systemjs/dist/system.src.js',
-      'zone.js/dist/*.js',
+      'zone.js/dist/**/*.*',
```

Comment:
Thought it would be better because of the sourcemaps. Maybe copying `js` and `js.map` might be enough though.